internal/cmd/pipeline/command_update.go

```diff
@@ -93,16 +93,9 @@ func (c *command) update(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if name == "" && description == "" && sqlFile == "" && len(secrets) == 0 && ksqlCluster == "" &&
-		!cmd.Flags().Changed("activation-privilege") && !cmd.Flags().Changed("update-schema-registry") {
-		return fmt.Errorf("one of the update options must be provided:" +
-			" `--name`," +
-			" `--description`," +
-			" `--ksql-cluster`," +
-			" `--sql-file`," +
-			" `--secret`," +
-			" `--activation-privilege`," +
-			" `--update-schema-registry`")
+	if err := errors.CheckNoOpUpdate(cmd.Flags(), "name", "description",
+		"ksql-cluster", "sql-file", "secret", "activation-privilege", "update-schema-registry"); err != nil {
```

Comment:
Let's prefer this when the list of flags gets too long for one line? (Alphabetizing the list would be nice too)
```suggestion
	flags := []string{
		"name",
		"description",
		"ksql-cluster",
		"sql-file",
		"secret",
		"activation-privilege",
		"update-schema-registry",
	}
	if err := errors.CheckNoUpdate(cmd.Flags(), flags...); err != nil {
```